kubeportal/k8s/k8s_sync.py

```diff
@@ -14,19 +14,18 @@
 from kubernetes import client
 from kubeportal.models.kubernetesnamespace import KubernetesNamespace
 from kubeportal.models.kubernetesserviceaccount import KubernetesServiceAccount
-from kubeportal.k8s.utils import load_config
 from kubeportal.k8s import ns_sync_utils as ns_utils
 from kubeportal.k8s import svca_sync_utils as svca_utils
 from kubeportal.k8s import kubernetes_api as api
+from kubeportal.k8s.kubernetes_api import core_v1, rbac_v1
```

Comment:
seems like core_v1 and rbac_v1 aren't used?

Reply:
Right, this is a remaining from the refactoring. Thanks.